clodius/tiles/bed2ddb.py

```diff
@@ -24,7 +24,7 @@ def get_2d_tileset_info(db_file):
     return tileset_info
 
 
-def get_1D_tiles(db_file, tile_ids):
+def get_1D_tiles(db_file, zoom, tile_x_pos, numx):
```

Comment:
`zoom` is referenced below, and the doc string suggests zoom is the second parameter.